manifests/init.pp

```diff
@@ -81,6 +81,7 @@
   $python_virtualenvs        = { },
   $python_pyvenvs            = { },
   $python_requirements       = { },
+  $python_dotfiles           = { },
```

Comment:
this needs to be validated. Please add `validate_hash($python_dotfiles)` to the manifest.